util/inc/leatherman/util/scoped_descriptor.hpp

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
The subdirectories were a way to simply make it clear that this particular type was specific to certain platforms (it was also in a `posix` namespace if memory serves).  It's simply to make it clear that this is not something you can include on every platform.  I'm not particular married to this convention, but I found it to be useful, personally.